app/models/job.ts

```diff
@@ -1,5 +1,5 @@
 import { pick } from 'lodash';
-import { IWithPagination } from 'knex-paginate'; // For types only
+import { IPagination, IWithPagination } from 'knex-paginate'; // For types only
```

Comment:
It looks like we should either use `IPagination` or `IWithPagination` everywhere. I assume `IWithPagination` since that is what was here before.

Reply:
Typescript wouldn't let me pass the pagination object to getPagingLinks without using `IPagination`. I'm not sure why James's code didn't complain when the type was `IWithPagination` but it may have to do with how he wrote his variable assignment?

```
let pagination;
    await db.transaction(async (tx) => {
      if (!req.context.isAdminAccess) {
        ({
          job,
          pagination,
        } = await Job.byUsernameAndRequestId(tx, req.user, jobID, false, page, limit));
      } else {
        ({ job, pagination } = await Job.byRequestId(tx, jobID, page, limit));
      }
    });
    if (job) {
      const urlRoot = getRequestRoot(req);
      const pagingLinks = getPagingLinks(req, pagination).map((link) => new JobLink(link));
```

Reply:
https://github.com/nasa/harmony/pull/42/commits/1b6f5b9c9b3b3053c02904160ec4c06b868ee656

Here's the new change. I can revert it if you have other thoughts.

Reply:
I just do what the Typescript gods tell me to do - this looks fine to me.